src/mscorlib/ref/Compat/mscorlib.cs

```diff
@@ -11469,6 +11694,12 @@ public sealed partial class NeutralResourcesLanguageAttribute : System.Attribute
         public string CultureName { get { return default(string); } }
         public System.Resources.UltimateResourceFallbackLocation Location { get { return default(System.Resources.UltimateResourceFallbackLocation); } }
     }
+    internal partial class PRIExceptionInfo
```

Comment:
@weshaggard asked me to add any types that were added to the coreclr's mscorlib reference assembly, even if they were internal, which is basically what this whole PR is doing.